[OSDOCS-14600] HCP for Openstack (#93726)
openshift/openshift-docs: hosted_control_planes/hcp-deploy/hcp-deploy-openstack.adoc

:_mod-docs-content-type: ASSEMBLY
[id="hcp-deploy-openstack"]
include::_attributes/common-attributes.adoc[]
= Deploying {hcp} on OpenStack
:context: hcp-deploy-openstack

toc::[]

You can deploy {hcp} with hosted clusters that run on {rh-openstack-first}.

> A _hosted cluster_ is an {product-title} cluster with its API endpoint and control plane that are hosted on the hosting cluster. The hosted cluster includes the control plane and its corresponding data plane. You can use the {mce-short} console or the `hcp` command-line interface (CLI) to create a hosted cluster.

Reviewer: nit: I think you've taken part of this from [here](https://docs.redhat.com/en/documentation/openshift_container_platform/4.14/html/hosted_control_planes/hosted-control-planes-overview) and started but not finished reworking it?

```suggestion
A _hosted cluster_ is an {product-title} cluster with its API endpoint and control plane hosted on a management cluster. The hosted cluster includes the control plane and its corresponding data plane. You can use the {mce-short} console or the `hcp` command-line interface (CLI) to create a hosted cluster.
```

You could also say _on a hosting cluster_, but IMO hosted and hosting are less clear than hosted and management.
